manifests/f/Famatech/AdvancedIPScanner/2.5.3850/Famatech.AdvancedIPScanner.installer.yaml

```diff
@@ -2,10 +2,20 @@
 
 PackageIdentifier: Famatech.AdvancedIPScanner
 PackageVersion: 2.5.3850
+Platform:
+- Windows.Desktop
+InstallerType: exe
+UpgradeBehavior: install
 Installers:
 - Architecture: neutral
+  Scope: user
+  InstallerType: inno
+  InstallerUrl: https://download.advanced-ip-scanner.com/download/files/Advanced_IP_Scanner_2.5.3850.exe
+  InstallerSha256: 87BFB05057F215659CC801750118900145F8A22FA93AC4C6E1BFD81AA98B0A55
```

Comment:
You need to define a custom installer switch for this to install to the user

Reply:
Is this the mode you are trying to add?
![image](https://user-images.githubusercontent.com/15158490/146459326-3aac8f23-bb41-47b1-91c5-062f395e998c.png)
You need to find the Installer variable for that, which might be tricky since its for Portable version and not /CURRENTUSER

Reply:
No. the original installer was setup with no scope. So when attempting to install with a scope set to machine it said no matching package was found. 

So I assumed no packages were required for user as that is the default scope if none is defined. My assumption may be incorrect. 

I have added some switches. I tested /CURRENTUSER and it did not install into appdata. It still installed in Program Files.

Reply:
Realistically there might not be a user scope available for this app. Perhaps I remove that section and just set the default scope to machine?

Reply:
https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/installer.md#scope
I think as long as we don't have the Portable switch we might have to go with machine only ye

Reply:
still running reading up on inno and seeing if there is a combination we can do to run in portable mode unattended. If not I'll change it machine.